index.html

```diff
@@ -35,6 +39,12 @@
 
     <link rel="icon" href="/images/favicon.ico" />
 
+    <link rel="manifest" href="./manifest.json" />
```

Comment:
This is the app manifest. I added some basic info and icons but you might wanna add more.